src/pages/admin-settings.tsx

```diff
@@ -0,0 +1,319 @@
+      <label className='block mb-4'>
+        Assigned Users:
+        <input
+          className='border border-gray-400 rounded w-full p-2'
+          type='text'
+          value={assignedUsers}
+          onChange={(e) => setAssignedUsers(e.target.value)}
+        />
+      </label>
+      <button
+        className='bg-purple-600 text-white py-2 px-4 rounded mr-2'
+        onClick={handleUpdateTask}
+      >
+        Update Task
+      </button>
+      <button
+        className='bg-gray-200 text-red-600 py-2 px-4 rounded'
+        onClick={handleCancel}
+      >
+        Cancel
+      </button>
+    </div>
+  );
+};
+
+const Users = () => {
+  const [displaySkills, setDisplaySkills] = useState<number | undefined>(undefined);
+  const [showModal, setShowModal] = useState(false);
+  const [currentTable, setCurrentTable] = useState(1);
+  const [modalContent, setModalContent] = useState<React.ReactNode>(null);
+  const [showSortOptions, setShowSortOptions] = useState(false);
+
+
+const handleMoreClick = (id: number) => {
+    if (displaySkills === id) {
+      setDisplaySkills(undefined);
+    } else {
+      setDisplaySkills(id);
+    }
+  };
+  
+
+  return (
+    <div className='bg-gray-100 min-h-screen'>
+      {showModal && (
+      <div className='fixed top-0 left-0 w-full h-full bg-black bg-opacity-50 flex justify-center items-center'>
+        <div className='bg-white p-4 rounded-lg'>
+          {modalContent}
+          </div>
+          </div>
+          )}
+      <div className='flex justify-between p-4'>
+        <h2>Admin Dashboard</h2>
+        <h2>Welcome Back, Nhi</h2>
+      </div>
+      <div className='p-4'>
+  <div className='flex justify-between mb-4'>
+    <div>
+      <button
+        className={`py-2 px-4 rounded ${
+          currentTable === 1 ? 'bg-purple-600 text-white' : 'bg-gray-200 text-purple-800'
+        }`}
+        onClick={() => setCurrentTable(1)}
+      >
+        Users
+      </button>
+      <button
+        className={`py-2 px-4 rounded ml-2 ${
+          currentTable === 3 ? 'bg-purple-600 text-white' : 'bg-gray-200 text-purple-800'
+        }`}
+        onClick={() => setCurrentTable(3)}
+      >
+        Task Queues
+      </button>
+    </div>
+    {currentTable === 3 && (
+      <button
+        className='bg-purple-600 text-white py-2 px-4 rounded mb-4'
+        onClick={() => {
+          setModalContent(<AdminEditTask setShowModal={setShowModal} />);
+          setShowModal(true);
+        }}
+      >
+        Create New Task Queue
+      </button>
+    )}
+  </div>
+        {currentTable === 3 && (
+             <div className='flex justify-end mb-4'>
+             <div className='relative'>
+               <button
+                 className='border border-gray-400 rounded px-4 mr-2 relative'
+                 onClick={() => setShowSortOptions(!showSortOptions)}
+                 style={{ height: '30px', paddingTop: '4px', paddingBottom: '4px' }}
+               >
+                  ▼Sort
```

Comment:
What do you think about using an icon instead?

https://react-icons.github.io/react-icons/search?q=chevron